### ISSUE TITLE
FastDecoderException Error

### ISSUE THREAD

**Issue by Nokrato** [PluginBugs/Issues-ItemsAdder#3109]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### What happened?

**Environment:**

**Minecraft Version:** 1.20.2
**Server Software:** Spigot-Paper 1.20.2
**BungeeCord Version:** Waterfall (latest version)
**Plugin Version:** ItemAdder v3.6.1-beta-r2
**Bungee Plugin Causing Issue:** BungeePackFix v1.1.4

**Description:**
When clicking the recipe book, a FastDecoderException error occurs, disrupting gameplay.

**Steps to Reproduce:**
Open the recipe book.

**Expected Behavior:**
The recipe should appear normally without causing any exceptions or errors.

### Steps to reproduce the issue

Open the recipe book.

### Server version

This server is running Paper version git-Paper-246 (MC: 1.20.2) (Implementing API version 1.20.2-R0.1-SNAPSHOT) (Git: c1ac983)
You are running the latest version
Previous version: git-Paper-234 (MC: 1.20.2)

### ItemsAdder Version

3.6.1-beta-r2

### ProtocolLib Version

5.1.1-SNAPSHOT-669

### LoneLibs Version

5.1.1-SNAPSHOT-669

**Comment by LoneDev6** [comment]
Please post the error or complete log otherwise I cannot identify the issue.

**Comment by Nokrato** [comment]
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/48636363/a4537704-66a9-4f4c-be2f-c14bd576f772)

https://paste.helpch.at/qofutitoze.makefile

Bungee error:
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/48636363/dc2f0d8b-8f9e-4996-b8b6-5c1e4c0666a4)

@LoneDev6

**Comment by LoneDev6** [comment]
Set `print-full-stacktraces: true` in your proxy configuration in order to get the full error in bungeecord console.

**Comment by Nokrato** [comment]
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/48636363/72264882-4ae9-46de-ad5c-badd3be008ce)
i wasn't able to locate that opinion can you tell me where its .... also i found out that by disabling this voice in the BungeePackFix plugin the server works fine  @LoneDev6

**Comment by LoneDev6** [comment]
@Nokrato  This just basically disables the plugin, so you can just remove BungeePackFix completely

**Comment by AfkF24** [comment]
This appears to be a rather annoying issue related to BungeePackFix
```
[18:26:14 ERROR]: [AfkF24] -> UpstreamBridge - exception caught Type: FastDecoderException
[18:26:14 INFO]: [AfkF24] disconnected with: FastDecoderException : Error decoding packet ID(24) Type(class dev.lone.bungeepackfix.bungee.packets.impl.ServerboundResourcePackPacket)
[18:26:14 INFO]: [AfkF24] -> UpstreamBridge has disconnected
```

**Comment by LoneDev6** [comment]
@AfkF24 which bungeecord version?
which spigot version?
which CLIENT version used to connect?

**Comment by AfkF24** [comment]
> @AfkF24 which bungeecord version?
> 
> which spigot version?
> 
> which CLIENT version used to connect?

Bungee 1.20.2, Paper 1.20.2, Client 1.20.2

**Comment by LoneDev6** [comment]
@AfkF24 can you make sure you updated Bungeecord to the latest version?
If you have ViaVersion, ViaBackwards or ProtocolLib please update them all too.

**Comment by AfkF24** [comment]
@LoneDev6 yeah can confirm that Bungee is up to date. No Via plugins and ProtocolLib build is [673](https://ci.dmulloy2.net/job/ProtocolLib/673/), so that should be fine as well
